.vib/grafana-mimir/goss/vars.yaml

```diff
@@ -0,0 +1,51 @@
+mimir:
+  dataDir: "/bitnami/grafana-mimir"
+  containerPorts:
+    http: 8080
+    gossipRing: 7946
+  gossipRing:
+    service:
+      ports:
+        http: 7946
+gateway:
+  enabled: true
+  service:
+    ports:
+      http: 80
+    type: LoadBalancer
+compactor:
+  service:
+    ports:
+      http: 8080
+distributor:
+  service:
+    ports:
+      http: 8080
```

Comment:
I'd use a completely different port for each deployment. Imagine the following scenario: "I did copy-paste in all of the deployments but forgot to change the service port and it is being set to `gateway.service.ports.http` in all of them. This would not be detected in this case.